rpcserver.go

```diff
@@ -49,6 +49,14 @@ import (
 	"github.com/decred/dcrutil"
 )
 
+// API version constants
+const (
+	jsonrpcSemverString = "1.0.0"
```

Comment:
Is there a reason for having this as a duplicate of the version info rather than constructing from the individual version parts?  Since it is a string not really much to gain from it being a 'const'.